task_1/andrii.lukiacnhykov/task-1/src/main/java/Classes/Commit/Commit.java

```diff
@@ -0,0 +1,131 @@
+package Classes.Commit;
+
+import Classes.Blob.Blob;
+import Classes.Parent.Parent;
+import Classes.Tree.Tree;
+import Classes.Type.Type;
+import org.apache.commons.codec.digest.DigestUtils;
+
+import java.text.SimpleDateFormat;
+import java.time.LocalDate;
+import java.util.Date;
+import java.util.Map;
+
+public class Commit extends Type
+{
+
+    private String tree;
+    private String parent;
+    private String hashcode = null;
+    private String message;
+    private LocalDate date;
+
+    public Commit() {
+
+    }
+
+    public Commit(String treeHashCode,String parentHashCode, String message) // String treeHashCode, String parentHashCode
```

Comment:
To understand where i am, and what other Constructions or methods i need to create